content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1341,6 +1341,19 @@ records the date (ISO 8601 format, UTC time zone) when the control plane detects
 that the auto-generated Secret has not been used for a specified duration
 (defaults to one year).
 
+### endpoints.kubernetes.io/managed-by
+
+Type: Label
+
+Example: `endpoints.kubernetes.io/managed-by: endpoint-controller`
+
+Used on: Endpoints
+
+The label is used to indicate the controller or entity that manages the
+Endpoints. The value `endpoint-controller` indicates an Endpoints object that
+was created automatically by Kubernetes for a Service with a
+{{< glossary_tooltip text="selectors" term_id="selector" >}}.
+
```

Comment:
```suggestion

{{< note >}}
The [Endpoints](/docs/reference/kubernetes-api/service-resources/endpoints-v1/) API is deprecated.
If you actively use the deprecated Endpoints API in your cluster, this label is still relevant.
{{< /note >}}

```

Reply:
"relevant" is a stretch... We don't expect people to write this label themselves, and they don't really get any benefit from reading it either.

I rewrote the doc to clarify that this is there for kubernetes-internal purposes.